src/Appwrite/Migration/Version/V21.php

```diff
@@ -177,7 +188,7 @@ protected function fixDocument(Document $document): Document
                 $document->setAttribute('scopes', []);
 
                 // Add size attribute
-                $document->setAttribute('specification', APP_FUNCTION_BASE_SPECIFICATION);
+                // $document->setAttribute('specification', APP_FUNCTION_BASE_SPECIFICATION); Uncomment after we merge variable runtimes specifications.
```

Comment:
Double check we also unset it in request filter in the other PR

Reply:
Good catch, added it